docs/README.md

```diff
@@ -1,6 +1,11 @@
-# Duke
+# Upcycle
 
-{Give product intro here}
+Hi there, 
+
+Welcome to Upcycle - a perfect desktop app dedicate for managing rental businesses. The nature of these businesses
+requires to deal with tons of data; therefore, noting down all information by notes and pen is not ideal for managers.
+It is developed for rental business managers, who can type fast to efficiently keep track all of their customers, items,
+and transactions via Command Line Interface with a single line of command.
 
 Useful links:
 * [User Guide](UserGuide.md)
```

Comment:
Here abt the input, it's a bit confusing for me.
Sine when reading a input, the control alrd should be passed to Ui and later return it.
![3](https://user-images.githubusercontent.com/71815678/198496413-9edbd6bb-5e3c-4937-b2ae-e7e4ea33ee6a.png)

For the second `input`, it never returns any control